ccd-definition/CaseEventToFields/CareSupervision/createOrder-nonprod.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "createOrder",
+    "CaseFieldID": "interimEndDate",
+    "PageFieldDisplayOrder": 3,
+    "DisplayContext": "MANDATORY",
+    "PageID": "OrderDateOfIssue",
+    "PageLabel": " ",
```

Comment:
These label fields are unnecessary, they are used to append information to the title of the page. If you had an entry `"PageLabel": "something"` for an event with the name `Event name` then the title on the page will appear as `something - Event name`. The default behaviour when the field is not included is the same as what you have here which is just making nothing appear before it.